src/frontend/metrics/generateMetricsReport.test.ts

```diff
@@ -1,38 +1,46 @@
 import * as path from 'node:path';
 import * as fs from 'node:fs';
-import generateMetricsReport from './generateMetricsReport';
+import {
+  generateMetricsReport,
+  getPercentageOfNetworkAvailability,
+} from './generateMetricsReport';
+import {generate} from '@mapeo/mock-data';
+import positionToCountries from './positionToCountries';
+import type {Observation} from '@mapeo/schema';
+import {addToSet} from './utils';
 
 type MetricsReportOptions = Parameters<typeof generateMetricsReport>[0];
 
 describe('generateMetricsReport', () => {
   const packageJson = readPackageJson();
+  const count = 10;
+  const n = Math.floor(Math.random() * count);
+  const observations: ReadonlyArray<Observation | Record<string, never>> = [
+    ...generate('observation', {count}).map((obs, idx) =>
+      // Manually add Machias Seal Island, disputed territory
+      idx === n ? {...obs, lat: 44.5, lon: -67.101111} : obs,
+    ),
+    {},
+  ];
```

Comment:
Would you mind adding Machias Seal Island to the end, rather than mixing it in?

```suggestion
  const observations: ReadonlyArray<Partial<Observation>> = [
    ...generate('observation', {count: 10}),
    // Machias Seal Island, disputed territory
    {lat: 44.5, lon: -67.101111},
    {},
  ];
```